packages/twenty-front/src/modules/activities/emails/components/EmailPreview.tsx

```diff
@@ -65,7 +78,7 @@ const StyledReceivedAt = styled.div`
 
 type EmailPreviewProps = {
   divider?: boolean;
-  email: Email;
+  email: TimelineMessage;
```

Comment:
nit: Not for now but TimelineMessage is a generic type from the BE but we might want to map this to an Email type here?